test classe navngivning ændret,

Ironfirst opened the pull request:
Movement wind class
lavet med test

Ironfirst committed:
test classe navngivning ændret,

IllegalArgumentException til oceanX og oceanY er lavet så
værdierne ikke kan være under 1 ved oprettelse

mikk7867 commented on src/main/java/sas1/project/demo/model/Wind.java:
```diff
@@ -19,6 +20,222 @@ public Wind(int windId, String windDirection, int windSpeed) {
+                this.windDirection="N";
+            }
+            else{
+                this.windDirection="NW";
```
direction går igen, er NW skal være NE

mikk7867 commented on src/main/java/sas1/project/demo/model/Wind.java:
```diff
@@ -19,6 +20,222 @@ public Wind(int windId, String windDirection, int windSpeed) {
+
+        if(currentDbWind=="S"){
+            if(directionChange==1){
+                this.windDirection="SW";
```
SW står to gange, den ene skal være NW

mikk7867 commented on src/main/java/sas1/project/demo/model/Wind.java:
```diff
@@ -19,6 +20,222 @@ public Wind(int windId, String windDirection, int windSpeed) {
+            }
+        }
+
+        if(currentDbWind=="S"){
```
skift afhængig af retning er omvendt

mikk7867 commented on src/main/test/sas1/project/demo/model/WindTest.java:
```diff
@@ -0,0 +1,85 @@
+        assertTrue(res);
+
+        wind.setWindDirection("S");
+
```
getNewWindAttributes skal køres her, ellers bliver vindretning ved med at være syd

mikk7867 commented on src/main/java/sas1/project/demo/model/Wind.java:
```diff
@@ -9,6 +9,7 @@
     private int windId;
```
hvad gør den her? windId

mikk7867 reviewed (changes requested):
se kommentarer

Ironfirst committed:
test classe navngivning ændret,

IllegalArgumentException til oceanX og oceanY er lavet så
værdierne ikke kan være under 1 ved oprettelse

Ironfirst committed:
test classe navngivning ændret,

IllegalArgumentException til oceanX og oceanY er lavet så
værdierne ikke kan være under 1 ved oprettelse

mikk7867 reviewed (approved):
ingen problemer
